Project3/styles.css

```diff
@@ -0,0 +1,111 @@
+input[type="radio"],
+input[type="checkbox"] {
+    -moz-appearance: none;
+    -webkit-appearance: none;
+    appearance: none;
+    border: 2px solid white;
+    height: 18px;
+    width: 18px;
+    display: inline-block;
+}
+
+/* font-family: 'BenchNine', sans-serif; */
+/* all width 800px 1024px 1120px (form) */
+body {
+    background-color: #163346;
+    align: auto;
+    padding: 16px;
+    margin: auto;
+    max-width: 300px;
+    min-width: 300px;
+    font-family: 'BenchNine', sans-serif;
+}
```

Comment:
I would expect the `body` and `box-sizing` rules to be the first thing in your stylesheet, since they are setting the defaults for everything that follows. After those baseline styles:
1. blank/bare elements without `id` or classnames
1. selectors with `class`es
1. selectors with `id`s
1. media queries

In this way, your stylesheet gets _more_ specific as it gets longer.